app/services/get_application_choices_waiting_for_candidate_decision.rb

```diff
@@ -0,0 +1,14 @@
+class GetApplicationChoicesWaitingForCandidateDecision
+  def self.call
+    ApplicationChoice
```

Comment:
As discussed, something like this to get the application forms that need chasing:

```
ApplicationForm.where(
  id: ApplicationChoice
    .where(status: :offer)
    .where('decline_by_default_at < ?', chase_candidate_time_limit)
    .select(:application_form_id)
).where.not(id: ChaserSent.candidate_decision_request.select(:chased_id))
```